src/components/dataset/query/sidebar/QuerySidebarPanel.jsx

```diff
@@ -2,15 +2,26 @@ import React from 'react';
 import _ from 'lodash';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { withStyles } from '@material-ui/core/styles';
 
 import Card from '@material-ui/core/Card';
 import Button from '@material-ui/core/Button';
 import Typography from '@material-ui/core/Typography';
 import HighlightOff from '@material-ui/icons/HighlightOff';
 import { applyFilters } from '../../../../actions';
 
+const styles = theme => ({
+  cardPadding: {
+    padding: '10px',
+  },
```

Comment:
nit: should we be using the theme spacing here?

Reply:
Good point